### ISSUE TITLE
Stop using Stackblitz

### ISSUE THREAD

**Issue by djmill2** [stackblitz/core#594]
I need to stop having this trigger at this time. I am sure it is a good program, but it is not what I want. How do I stop it from popping up when I want to use a different IDE.

**Comment by mlc-mlapis** [comment]
@djmill2 ... about which `trigger` do you talk? What is the different IDE?

**Comment by mlc-mlapis** [comment]
@djmill2 ... not sure what happened. I guess that the permission  for StackBlitz is related to access to GitHub only. And also we are talking about online web editors so there is some URL on the beginning on which your are clicking ... and I can't imagine the the link to Plunker example is somehow redirected to Stackblitz.

**Comment by tiliev** [comment]
@djmill2, can you send us a screenshot of this so-called trigger?

**Comment by djmill2** [comment]
Other than the fact that the course examples would open stackBlitz rather
than Plunker, is kind of moot now. I tried running these examples on my
desktop as opposed to my laptop, and found that on the desktop the
stackblitz would run the examples without issue , which it would not do on
my laptop. So my laptop has some issues that I will have to address.
I would still like to figure out how to get StackBlitz removed from my
apparent selection, as the preferred online IDE, so that I can. run
whatever online IDE I choose to run rather than having StackBlitz jump out
first. On my desktop, the first time I opened one of the examples plunker
did start first, then after that StackBlitz took over.
